src/app/main/component/shared/components/warning-pop-up/warning-pop-up.component.ts

```diff
@@ -6,17 +6,17 @@ import { takeUntil } from 'rxjs/operators';
 @Component({
   selector: 'app-warning-pop-up',
   templateUrl: './warning-pop-up.component.html',
-  styleUrls: ['./warning-pop-up.component.scss'],
+  styleUrls: ['./warning-pop-up.component.scss']
 })
 export class WarningPopUpComponent implements OnInit, OnDestroy {
   public popupTitle: string;
   public popupSubtitle: string;
   public popupConfirm: string;
   public popupCancel: string;
   public closeButton = './assets/img/profile/icons/cancel.svg';
-  private destroyed$: ReplaySubject<any> = new ReplaySubject<any>(1);
```

Comment:
there is no need to remove private
use // @ ts-ignore instead

Reply:
done